packages/esm-active-visits-app/src/active-visits-widget/active-visits.component.tsx

```diff
@@ -187,125 +153,128 @@ const ActiveVisitsTable = () => {
         </div>
-        <Search
-          labelText=""
-          placeholder={t('filterTable', 'Filter table')}
-          onChange={handleSearch}
-          size={isDesktop(layout) ? 'sm' : 'lg'}
-        />
-        <DataTable
-          rows={results}
-          headers={headerData}
-          size={isDesktop(layout) ? 'sm' : 'lg'}
-          useZebraStyles={activeVisits?.length > 1 ? true : false}>
-          {({ rows, headers, getHeaderProps, getTableProps, getRowProps, getExpandHeaderProps }) => (
-            <TableContainer className={styles.tableContainer}>
-              <Table className={styles.activeVisitsTable} {...getTableProps()}>
-                <TableHead>
-                  <TableRow>
-                    <TableExpandHeader enableToggle {...getExpandHeaderProps()} />
-                    {headers.map((header) => (
-                      <TableHeader {...getHeaderProps({ header })}>{header.header}</TableHeader>
-                    ))}
-                  </TableRow>
-                </TableHead>
-                <TableBody>
-                  {rows.map((row, index) => {
-                    const currentVisit = activeVisits.find((visit) => visit.id === row.id);
+        <div className={styles.backgroundDataFetchingIndicator}>
+          <span>{isValidating ? <InlineLoading /> : null}</span>
+        </div>
+      </div>
+      <Search
+        labelText=""
+        placeholder={t('filterTable', 'Filter table')}
+        onChange={handleSearch}
+        size={isDesktop(layout) ? 'sm' : 'lg'}
+      />
+      <DataTable
+        isSortable
+        useStaticWidth
+        rows={results}
+        headers={headerData}
+        sortRow={sortRow}
+        size={isDesktop(layout) ? 'sm' : 'lg'}
+        useZebraStyles={activeVisits?.length > 1}>
+        {({ rows, headers, getHeaderProps, getTableProps, getRowProps, getExpandHeaderProps }) => (
+          <TableContainer className={styles.tableContainer}>
+            <Table className={styles.activeVisitsTable} {...getTableProps()}>
+              <TableHead>
+                <TableRow>
+                  <TableExpandHeader enableToggle {...getExpandHeaderProps()} />
+                  {headers.map((header) => (
+                    <TableHeader {...getHeaderProps({ header })}>{header.header}</TableHeader>
+                  ))}
+                </TableRow>
+              </TableHead>
+              <TableBody>
+                {rows.map((row, index) => {
+                  const currentVisit = activeVisits.find((visit) => visit.id === row.id);
 
-                    if (!currentVisit) {
-                      return null;
-                    }
+                  if (!currentVisit) {
+                    return null;
+                  }
 
-                    const patientChartUrl = '${openmrsSpaBase}/patient/${patientUuid}/chart/Patient%20Summary';
+                  const patientChartUrl = '${openmrsSpaBase}/patient/${patientUuid}/chart/Patient%20Summary';
```

Comment:
You need to redirect to just `/chart` and the dashboard to redirect, let it be Patient Summary or any other is handled by the patient chart itself.